release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -19,13 +19,13 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 {product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cloud-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
 
-{product-title} {product-version} is supported on {op-system-base-full} 7.7 or later, as well as {op-system-first} 4.6.
+{product-title} {product-version} is supported on {op-system-base-full} 7.9 or later, as well as {op-system-first} 4.6.
```

Comment:
Since these are 4.7 RNs, shouldn't this be `...as well as {op-system-first} 4.7.`?

Reply:
Yep - fixing that is what uncovered this. See #31181, ready to merge tomorrow to allow a couple days for the change notification to be absorbed :)